src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSGapicSurfaceDocTransformer.java

```diff
@@ -51,7 +51,7 @@
   public List<ViewModel> transform(ApiModel apiModel, GapicProductConfig productConfig) {
     Model model = ((ProtoApiModel) apiModel).getProtoModel();
     ImmutableList.Builder<ViewModel> surfaceDocs = ImmutableList.builder();
-    for (ProtoFile file : new ProtoFileView().getProtoFiles(productConfig)) {
+    for (ProtoFile file : new ProtoFiles().getProtoFiles(productConfig)) {
```

Comment:
Should we use ProtoFiles statically?

Reply:
yep, I overlooked this when I did the conversion.